couchbase/kv/v1/kv.proto

```diff
@@ -59,10 +64,14 @@ message GetRequest {
   string collection_name = 3;
   string key = 4;
   repeated string project = 5;
+  optional CompressionEnabled compression = 6;
 }
 
 message GetResponse {
-  bytes content = 1;
+  oneof content {
+    bytes content_uncompressed = 1;
```

Comment:
Note that while this indicates that the change is not backwards compatible, in practice it actually is backwards compatible since you will always receive field 1 unless you have explicitly specified the compression mode in the request, which implies the client will be aware of both the compressed or uncompressed content.